core/Enclave/self_test/rsa_test.cpp

```diff
@@ -449,19 +449,19 @@ static void RSA_create_key(RSA *key, const char *n, const char *e, const char *d
                  privateExponent);
 }
 
-static ehsm_padding_mode_t get_paddingmode(int paddingmode)
+static uint32_t get_paddingmode(int paddingmode)
 {
```

Comment:
duplicate? seems there're getDigestMode and getPaddingMode?

Reply:
it only use in test, change the name to getDigestMode  and getPaddingMode

Reply:
can we reuse the original one? such as use "extern"?

Reply:
ok